### ISSUE TITLE
Clarify difference between `adonis serve` with and without --dev flag

### ISSUE THREAD

**Issue by benknight** [adonisjs/adonis-cli#188]
`adonis serve --help` shows:

```
--dev Start development server
```

As far as I can gather what this actually means is it will watch files for changes while `adonis serve` (without the flag) won't

**Comment by thetutlage** [comment]
Yes. The `--dev` flag means a file watcher will monitor for file changes. I am happy to accept the PR to improve the flag description :)
